### ISSUE TITLE
Starting page 

### ISSUE THREAD

**Issue by Cherboi** [Cherboi/Fish-Quiz#2]
create a starting page that includes highscore and Start Quiz button

**Comment by Jon2328** [comment]
Hi! may I help on this?
Do you have mock or anything that you would like me to follow?

**Comment by Cherboi** [comment]
Yes I would love help  On Sun, Jan 30, 2022, 11:49 PM Jon2328 ***@***.***> wrote:Hi! may I help on this?Do you have mock or anything that you would like me to follow?—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by Jon2328** [comment]
I have submitted the pull request, hope it helps :)
